web/components/widgets/tooltip.tsx

```diff
@@ -16,14 +16,15 @@ import { ReactNode, useRef, useState } from 'react'
 // See https://floating-ui.com/docs/react-dom
 
 export function Tooltip(props: {
-  text: string | false | undefined | null
+  text: string | false | undefined | null | ReactNode
   children: ReactNode
   className?: string
   placement?: Placement
   noTap?: boolean
   noFade?: boolean
+  externalOpen?: boolean
```

Comment:
@sipec I added this to basically override the tooltip disappearing upon hovering over it, let me know if you have a less hacky way to do it lol

Reply:
yes, you can use SafePolygon: https://floating-ui.com/docs/usehover#handleclose
you use it like `useHover(context, { mouseOnly: noTap, handleClose: safePolygon() }),`